demosplan/DemosPlanCoreBundle/ResourceTypes/SimilarStatementSubmitterResourceType.php

```diff
@@ -70,7 +73,11 @@ function (ProcedurePerson $submitter, array $similarStatements): array {
 
                     return [];
                 }),
-            $this->createToOneRelationship($this->procedure)->addConstructorBehavior(new RequiredToOneRelationshipConstructorBehaviorFactory(
+            // FIXME : have to check this change, there is no RequiredToOneRelationshipConstructorBehavior class that's why i used
```

Comment:
What about this fixme?

Reply:
@mussbach i checked it one more time and it's fixed here https://github.com/demos-europe/demosplan-core/pull/3515/commits/6b95fd57f3d2fdf55ae0fb0d6c4d76794bb3cbd1 . 
The 'RequiredToOneRelationshipConstructorBehavior' does not exists anymore and we have to use now directly the 'RequiredRelationshipConstructorBehavior' which is already adjusted ( see please https://github.com/demos-europe/edt-access-definitions/blob/0.24.42/src/Wrapping/PropertyBehavior/Relationship/ToOne/Factory/RequiredToOneRelationshipConstructorBehaviorFactory.php ).
I tested it too and it worked correctly ( Statement list -> Details and replies -> Edit -> then add or edit Other submitters of the same statement ). I think we use this resource type ( to update and add ) only there